### ISSUE TITLE
Write tests for link add/edit/remove audible feedback

### ISSUE THREAD

**Issue by tofumatt** [WordPress/gutenberg#11006]
We should add tests for #10795 to make sure the link speech commands are working as intended and not regressing, as it might be easy to miss in regular usage.

The patch was an easy win but @iseulde [pointed out I should have asked for tests](
https://github.com/WordPress/gutenberg/pull/10795#issuecomment-432552060), so I'm assigning myself to write them as penance 😆

If anyone else wants to though feel free and I'll review them! 😉

**Comment by gziolo** [comment]
I doesn't look like @tofumatt is actively working on it. This one needs a developer :)

**Comment by Mamaduka** [comment]
Can we close this? The last activity was four years ago. The link format is stable, and I've not seen any reports of audible feedback regressing.
